src/FullBuild/Program.fs

```diff
@@ -49,12 +49,12 @@ let tryMain argv =
     // view
     | Command.AddView viewInfo -> Commands.View.Add viewInfo
     | Command.DropView viewInfo -> Commands.View.Drop viewInfo.Name
-    | Command.ListViews -> Commands.View.List ()

```

Comment:
If you think dynamic views should be saved then all view command like create/build/test etc should be duplicated in "static" mode, but I don't think it's a good idea so I freez it for now